capstone/src/main/webapp/script.js

```diff
@@ -126,4 +126,33 @@ function getRegMessage() {
   }
   const node = document.importNode(template.content, true);
   document.body.appendChild(node);  
+}
+
+function showEdit() {
+  document.getElementById('description').contentEditable = 'true';
+  document.getElementById('website').contentEditable = 'true';
+  document.getElementById('officers-list').contentEditable = 'true';
+  document.getElementById('edit-button').hidden = 'true';
+  document.getElementById('edit-form').removeAttribute('hidden');
+}
+
+/** Store edited content from club page */
+function saveClubChanges() {
+  const newDesc = document.getElementById("description").innerHTML;
+  const newWebsite = document.getElementById("website").innerHTML;
+  var newOfficers = [];
+
+  const list = document.getElementById('officers-list');
+  const officerList = list.getElementsByTagName('li');
+  for (var i = 0; i < officerList.length; i++) {
+    console.log(officerList[i].innerText);
+    newOfficers.push(officerList[i].innerText);
+  }
+  
+  document.getElementById('new-desc').value = newDesc;
+  document.getElementById('new-web').value = newWebsite;
+  document.getElementById('new-officers').value = newOfficers;
+  document.getElementById('name').value = document.getElementById('club-name').innerHTML;
+  document.forms['edit-form'].submit();
+  alert("Changes saved!");
```

Comment:
nit: use single quotes for JS (unless double quotes are needed for alert statements?)